core/Archive.php

```diff
@@ -956,4 +970,13 @@ public function forceFetchingWithoutLaunchingArchiving()
     {
         $this->forceFetchingWithoutLaunchingArchiving = true;
     }
+
+    private function getRequestedReport(): ?string
```

Comment:
This always returns `null` unless triggered from CLI, is that correct? Should we perhaps think of a better name? `getCliRequestedReport` or `getCronRequestedReport` or so?

Reply:
Right now setting a requested report is only used/allowed when using cron archiving. That might not always be the case, and the specific rules about when an archiving feature is allowed to be used is considered separate from the archiving process itself. That said, I have no strong opinions on what this private method should be named. If you/the core team would rather it be named something else, let me know (w/ the specific name change).